completion.go

```diff
@@ -47,7 +47,12 @@ func (r *CompletionService) NewStreaming(ctx context.Context, body CompletionNew
 		err error
 	)
 	opts = append(r.Options[:], opts...)
```

Comment:
I recommend combining them into a single append that handles all options to improve readability.
```suggestion
	opts = append(r.Options[:], opts..., 
		option.WithJSONSet("stream", true), 
		// Streaming response returns a text/event-stream content type. To comply with
		// the SSE spec, we need to set the Accept header explicitly.
		option.WithHeader("Accept", "text/event-stream"),
	)
```

Reply:
I believe var length `opts...` cannot be bind together with an individual items this case (options.WithHeader)?